packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -109,9 +114,22 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
         sharedObjectRegistry: ISharedObjectRegistry,
         existing: boolean,
     ): FluidDataStoreRuntime {
-        return new FluidDataStoreRuntime(context, sharedObjectRegistry, existing);
+        return new FluidDataStoreRuntime(
+            context,
+            sharedObjectRegistry,
+            existing,
+            async (rt) => requestFluidObject(rt, "/"));
+    }
+
+    /**
+     * {@inheritDoc @fluidframework/datastore-definitions#IFluidDataStoreRuntime.entrypoint}
+     */
+    public get entrypoint(): IFluidHandle<FluidObject> | undefined {
+        return this.handle;
     }
 
+    private readonly handle?: IFluidHandle<FluidObject>;
```

Comment:
we don't really need handle and entrypoint, so i think we should just make entrypoint public readonly and only use that